specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/ResourceHealth.json

```diff
@@ -1231,6 +1689,33 @@
         }
       }
     },
+    "PotentialOutageErrorResponse": {
```

Comment:
Have you consider adding a reference to [ErrorResponse](https://github.com/Azure/azure-rest-api-specs/blob/dab488ef4572b30dee885044331326f8d1c8d329/specification/common-types/resource-management/v2/types.json#L313) from common-types?

Reply:
addressed.

Reply:
Doesn't look like it to me. You're still declaring your own.

---
In reply to: [551385095](https://github.com/Azure/azure-rest-api-specs/pull/12206#discussion_r551385095) [](ancestors = 551385095)